test/Main.hs

```diff
@@ -49,18 +50,12 @@ parseTemplate
       , "{}"
       ]
 
-    mkAccepted :: (Text, Paths.Template) -> TestTree
-    mkAccepted (input, expected)
-      = testCase (convertText ("accepts: " <> show input))
-      $ pure expected @=? parseJSON input
-
-    mkRejected :: Text -> TestTree
-    mkRejected input
-      = testCase (convertText ("rejects: " <> show input))
-      $ Left ("Error in $: invalid template path: " <> show input) @=? parseJSON input
-
-    parseJSON :: Text -> Either String Paths.Template
-    parseJSON = JSON.parseEither JSON.parseJSON . JSON.String
+    mkRejected' :: Text -> TestTree
```

Comment:
@mbj It seems a bit odd to me that `mkRejected'` is using `mkRejected` and not the other way round. Isn't it HS convention that the "prime versions" are typically local/private helpers?